### DISCUSSION TITLE
how to filter date string

### DISCUSSION THREAD

**Discussion by yoedusvany** [doug-martin/nestjs-query#1513] [General]
Hi, how can I filter by date field when I'm receiving a string like this:

filter: {dateField: {operador: '2022-10-10'}}

I need to filter documents by "gte" date

Any idea?

**Comment by smolinari** [comment]
@yoedusvany - You'd need to convert it into something the database you are using can work with. 

Scott

**Reply by yoedusvany** [reply]
Do you suggest to change the database schema for the date fields?

**Reply by smolinari** [reply]
@yoedusvany - What do you mean by "change"?

Scott

**Reply by yoedusvany** [reply]
data type for the field

**Reply by smolinari** [reply]
It would need to be a date type. Not sure what you would want to change it to. 

Scott
